Services/communication_service.h

```diff
@@ -29,6 +29,41 @@ struct temp_utc {
   uint32_t sensor_temperature;
 };
 
+struct Sband_config {
+  uint32_t S_freq;
+  uint32_t S_paPower;
+};
+
+struct Sband_PowerAmplifier {//must be uint8_t
```

Comment:
These structs will need to be packed:
`static stuct Sband_structure __attribute__((packed)) {...}`

Reply:
maybe do `typedefs`